EIPS/eip-3023.md

```diff
@@ -0,0 +1,194 @@
+---
+eip: 3023
+title: Contract Wallet Detection
+author: Ricardo Guilherme Schmidt (@3esmit)
+discussions-to: https://github.com/ethereum/EIPs/pull/3023
+status: Draft
+type: Informational
+created: 04-10-2020
```

Comment:
```suggestion
2020-10-04
```
ISO-8601

Reply:
Ah, great catch. I used my native format instead of the universal format. Thank you.